### PR TITLE
Milestone 2: Add aboutme and works section

### PR DESCRIPTION
**Pull request by isume295**
In this milestone I:
-Add about me page
-Add workers page
-Add contact me page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -47,6 +47,190 @@ <h2 class="headingTxt">I’m Sumeya. Glad to see you!</h2>
...
+      </section>
+
+      <section id="works">
+        <div class="worksGridContainer">
+         <div class="worksFlexContainer">
+          <div class="workImgContainer">
+           <img src="work1.png" alt="">
+          </div>
+          <div class="workDetailContainer">
+           <h1 class="projecTitle txtWrapper">Tonic</h1>
```
Comment (huseyinbiyik) [inline index.html]: Good job with using a heading tag. But you should use your h1 tag only once on a page. For this project, I am Sumeye, Glad to see you needs to be wrapped with h1 tag. All the others can use hx elements hierarchially. Please fix the related area on each card. Heading elements are important to configurate the importance of element on the page and they have a crucial role on SEO and accessibility onf your web pages.

File: index.html [hunk 2]
```diff
@@ -47,6 +47,190 @@ <h2 class="headingTxt">I’m Sumeya. Glad to see you!</h2>
...
+         </div>
+        </div>
+      </section>
+
+      <section id="about">
+        <div class="aboutMainContainer">
+          <div class="AboutContentContainer">
+            <h2 class="headingTxt">About Myself</h2>
+            <p class="paragraph">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <a class="connectMeLink" href="">LET’S CONNECT</a>
```
Comment (huseyinbiyik) [inline index.html]: You are right, `Let's connect` looks like a link here but actually it is a heading element of the social links. Kindly use hierarchically correct hx elements for it to make the page more semantic.

File: index.html [hunk 3]
```diff
@@ -47,6 +47,190 @@ <h2 class="headingTxt">I’m Sumeya. Glad to see you!</h2>
...
+            <div class="socialMediaContainer">
+                
+                <a href=""><svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (huseyinbiyik) [inline index.html:178]: Kindly use semantic `ul` element for wrapping the social media container and use `li` for each `a` element which include the link of social medium.

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi @sumeya8 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Grid and flexbox are used very well 👍 
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- Commit messages are meaningful and written in imperative tense 📝
- PR description is descriptive 📝

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — APPROVED** [review]
# STATUS: APPROVED ♻️♻️
Hi @isume295 

To highlight 👍🏽
✔️ Good semantic implementation
✔️ No linter errors

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**